### PR TITLE
[feature] Auto login

### PR DESCRIPTION
**Pull request by bastej**
- storing user data in async storage
- auto login into app if user data are saved in storage

### CODE REVIEW

File: redux/auth/saga.ts [hunk 1]
```diff
@@ -31,6 +36,10 @@ export function* loginSaga({ payload: { email, password } }: LoginAction) {
     const { idToken: token, localId: userId } = response;
 
     yield put(loginSuccess(token, { name: email, id: userId }));
+    yield AsyncStorage.setItem(
```
Comment (k-bialucha) [inline redux/auth/saga.ts:39]: Please use `yield call(AsyncStorage.setItem, ...)` (here and below)

File: redux/auth/saga.test.ts [hunk 2]
```diff
@@ -52,6 +53,8 @@ describe(`${DOMAIN_NAME}/saga`, () => {
           },
         })
       );
+      //
```
Comment (k-bialucha) [inline redux/auth/saga.test.ts:56]: Empty comment  - please remove.

File: redux/auth/saga.test.ts [hunk 3]
```diff
@@ -109,6 +113,8 @@ describe(`${DOMAIN_NAME}/saga`, () => {
           },
         })
       );
+      // TODO
```
Comment (k-bialucha) [inline redux/auth/saga.test.ts:116]: Please remove if it's not necessary.

File: redux/auth/saga.ts [hunk 4]
```diff
@@ -154,9 +175,39 @@ export function* loginWithGoogleSaga() {
   }
 }
 
+export function* tryAutoLoginSaga() {
+  try {
+    const jsonUserData = yield AsyncStorage.getItem('userData');
```
Comment (k-bialucha) [inline redux/auth/saga.ts:180]: Please use `yield call(AsyncStorage.getItem, ...)`.

**Comment by bastej** [comment]
> Requires small changes.

Done :+1:

**Pull request merged**